bitwarden_license/src/Portal/Startup.cs

```diff
@@ -46,6 +46,15 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<ICurrentContext, CurrentContext>((serviceProvider) =>
                 serviceProvider.GetService<EnterprisePortalCurrentContext>());
 
+            // Fido2
```

Comment:
Doesn't this duplicate the changes from #1248? See line 74 (of the changed version) below.